discord/commands.py

```diff
@@ -120,11 +123,24 @@ def __init__(self, type, /, **kwargs):
         self.type = type
         self.required = kwargs.pop("required", False)
         self.choices = list(i for i in kwargs.pop("choices", list()))
+    def to_dict(self):
+        return {
+            "name":self.name,
+            "description":self.description,
+            "type":int(self.type),
+            "required":self.required,
+            "choices":[c.to_dict() for c in self.choices]
+        }
     
 class OptionChoice:
     def __init__(self, name, value=None):
         self.name = name
         self.value = value or name
+    def to_dict(self):
+        return {
+            "name":self.name",
```

Comment:
```suggestion
            "name": self.name,
```

Reply:
huh

Reply:
@Middledot you left a " and missed a space lol